src/test/java/com/karankumar/bookproject/backend/service/AuthorServiceTest.java

```diff
@@ -0,0 +1,81 @@
+package com.karankumar.bookproject.backend.service;
+
+import com.karankumar.bookproject.annotations.IntegrationTest;
+import com.karankumar.bookproject.backend.entity.Author;
+import com.karankumar.bookproject.backend.repository.AuthorRepository;
+import lombok.extern.java.Log;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+@Log
+@IntegrationTest
+public class AuthorServiceTest {
+
+    private final AuthorService sut;
+    private final BookService bookService;
+    private final AuthorRepository authorRepository;
+
+    @Autowired
+    public AuthorServiceTest(AuthorService sut, BookService bookService, AuthorRepository authorRepository) {
```

Comment:
I don't think we need to autowire the `AuthorRepository`. We can call `deleteAll()` on the `AuthorService` instance